guidelines/goals.md

```diff
@@ -0,0 +1,69 @@
+# Goals and Non-Goals
+
+This document defines the purpose of the Message Format Working Group (MFWG)
+and informs the decisions about the scope and the priorities of its efforts.
+
+## Goals
+
+The primary task of the MFWG is to develop an industry standard for the
+representation of localizable message strings. The design goals are listed
+below.
+
+ 1. Express grammatical features, such as plurals, genders, and inflections.
+
+ 2. Express other variance in translation, due to linguistic and regional
+    features, the presentation media, context, circumstance, and other factors.
+
+ 3. Express formattable data, such as numbers, dates, currencies, and units.
+
+ 4. Represent structured data alongside translations, such as markup, comments,
+    and metadata.
+
+ 5. Ensure compatibility with the localization roundtrip workflows.
```

Comment:
```suggestion
 5. Be compatible with localization roundtrip workflows.
```

I think something like this line should stay as a goal, to keep us linked to the existing world. Perhaps expressing it slightly differently would help it fit better with the others?

Reply:
@DavidFatDavidF I think you have a better grasp on what this goal entails than I do. What do you think about the proposed phrasing?

Reply:
I think that the meaning of this one is that all and any features of the new message format should be capable of L10n roundtrip.
I kind of think that the word workflow(s) is superfluous here..
Maybe the wording should be something like
5. Be capable of localization roundtrip

Compatibility probably survives here from the time when this goal mentioned the XLIFF standard that you could comply with..

Reply:
Thank you! I think your suggestion is in line with @eemeli's and I like the simple wording too. I'll update the document.